azure-templates/deploy-steps.yml

```diff
@@ -51,11 +51,17 @@ parameters:
   - name: 'containerInstanceSubnet'
     type: string
     default: 'azure-devops'
+
+  # the branch, tag or commit to deploy
+  - name: 'git_reference'
+    type: string
+    default: $(Build.SourceVersion)
     
 steps:
   - template: ./make-build-steps.yml
     parameters:
       make: predeploy_build
+      git_reference: ${{ parameters.git_reference }}
```

Comment:
We can improve deploy step downloading the release from github instead make the third build.
Maybe this will be done in next PR.

Reply:
Next PR +1